src/main/java/com/brainpix/post/dto/GetIdeaListDto.java

```diff
@@ -0,0 +1,53 @@
+package com.brainpix.post.dto;
+
+import java.util.List;
+
+import com.brainpix.post.entity.PostAuth;
+import com.brainpix.post.entity.idea_market.IdeaMarketType;
+import com.brainpix.post.enums.SortType;
+import com.brainpix.profile.entity.Specialization;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+public class GetIdeaListDto {
+
+	@NoArgsConstructor
+	@Getter
+	@Setter
+	public static class Request {
+		private IdeaMarketType type;    // 아이디어 타입
+		private String keyword;        // 검색 키워드
+		private Specialization category;  // 카테고리
+		private Boolean onlyCompany;  // 기업 공개 제외/기업 공개만 보기
+		private SortType sortType;    // 정렬 기준
+	}
+
+	@Builder
+	@Getter
+	public static class Response {
+		List<ResponseData> responseDataList;    // 결과 값 리스트
```

Comment:
프런트 분들이 어떤 객체인지 모를수도 있고 추후 리팩토링시 혼동될수도 있을거 같아서 저는 어떤 역할의 객체인지 명확하게 네이밍 하는 편입니다 참고 부탁드려요!
```suggestion
		List<IdeaDetail> ideaDetailList;    // 아이디어 세부정보 리스트
```